concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1098,6 +1100,35 @@ jobs:
                 export CONFIG="$(pwd)/test-config/config.json"
                 ./paas-cf/tests/acceptance-tests/run_tests.sh
 
+  - name: bosh-cli-test
+    plan:
+      - aggregate:
+          - get: pipeline-trigger
+            passed: ['cf-deploy']
+            trigger: true
+          - get: paas-cf
+            passed: ['cf-deploy']
+          - get: concourse-manifest
+      - task: connect-to-bosh-cli
+        config:
+          platform: linux
+          image: docker:///governmentpaas/curl-ssl
```

Comment:
Can we use https://hub.docker.com/r/governmentpaas/self-update-pipelines/ instead?